samples/attested_tls/server/enc/openssl_server.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <arpa/inet.h>
+#include <openenclave/enclave.h>
+#include <openssl/evp.h>
+#include <openssl/ssl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include "../../common/openssl_utility.h"
+
+extern "C"
+{
+    int setup_tls_server(char* server_port, bool keep_server_up);
```

Comment:
```suggestion
    int set_up_tls_server(char* server_port, bool keep_server_up);
```
nit: Bad grammar in function name.  Suggest either `set_up_tls_server` or `initialize_tls_server` or `tls_server_setup`.